sonar-db/src/test/java/org/sonar/db/charset/SqlExecutorTest.java

```diff
@@ -73,13 +60,12 @@ public void executeSelect_executes_PreparedStatement() throws Exception {
   @Test
   public void executeUpdate_executes_PreparedStatement() throws Exception {
     dbTester.executeInsert(USERS_DB_TABLE, ImmutableMap.of(LOGIN_DB_COLUMN, "the_login", NAME_DB_COLUMN, "the name"));
-    dbTester.commit();
 
     try (Connection connection = dbTester.openConnection()) {
       underTest.executeUpdate(connection, "update users set " + NAME_DB_COLUMN + "='new name' where " + LOGIN_DB_COLUMN + "='the_login'");
-      connection.commit();
     }
-    Map<String, Object> row = dbTester.selectFirst("select " + NAME_DB_COLUMN + " from users where " + LOGIN_DB_COLUMN + "='the_login'");
```

Comment:
this fix is not enough for mysql as the returned key is `name` but not `NAME` (see line 69). Not a problem so far, I'm improving DbTester#selectFirst() in another branch.